tests/test_testing_cli.py

```diff
@@ -167,7 +167,8 @@ def start_test(runner: CliRunner, conf: ServerConfig, options: List[str] = []) -
 
 
 def start_target_server(runner: CliRunner, options: List[str] = []) -> ServerConfig:
-    conf = ServerConfig()
+    conf = ServerConfig(host="0.0.0.0")
```

Comment:
Why is this one hardcoded? don't we pass in port as well, or is this a different one

Reply:
This config is only used for this test.
I changed the base_url to "http://127.0.0.1:{port}", we need change host as well (host is default to "localhost"), instead of changing to 127.0.0.1, I just change it to 0.0.0.0 to make sure it is accessible by the base_url.